Kernel/DoubleBuffer.cpp

```diff
@@ -17,22 +17,36 @@ inline void DoubleBuffer::compute_lockfree_metadata()
     m_space_for_writing = m_capacity - m_write_buffer->size;
 }
 
+OwnPtr<DoubleBuffer> DoubleBuffer::try_create(size_t capacity)
+{
+    auto storage = KBuffer::try_create_with_size(capacity * 2, Region::Access::Read | Region::Access::Write, "DoubleBuffer");
+    if (!storage)
+        return {};
+
+    return AK::adopt_own_if_nonnull(new DoubleBuffer(capacity, storage.release_nonnull()));
```

Comment:
I'm not 100% up-to-date with allocations in the kernel, but shouldn't this be using a `nothrow` new?
unless the situation with that whole thing has changed since I last looked

Reply:
You are correct, I just missed it here. Thanks for catching!